<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnippetDialog</class>
 <widget class="QWidget" name="SnippetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>347</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="nameEdit">
              <property name="trapReturnKey">
              <bool>true</bool>
              </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="groupWidget" native="true">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabelGroup">
        <property name="text">
         <string comment="Group to which the snippet belongs.">Group:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="groupBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel">
        <property name="text">
         <string>&amp;Snippet:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>snippetText</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KTextEdit" name="snippetText">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="keyWidgetLabel">
        <property name="text">
         <string>Sh&amp;ortcut:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>keyWidget</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KKeySequenceWidget" name="keyWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KKeySequenceWidget</class>
   <extends>QWidget</extends>
   <header>kkeysequencewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
